spec/PhpSpec/Listener/CollaboratorMethodNotFoundListenerSpec.php

```diff
@@ -160,7 +160,7 @@ function it_warns_if_a_method_name_is_wrong(
         ConsoleIO $io,
         NameChecker $nameChecker
     ) {
-        $exception = new MethodNotFoundException('Error',DoubleOfInterface::class, 'throw');
+        $exception = new MethodNotFoundException('Error', new DoubleOfInterface(), 'throw');
```

Comment:
There are other instanciation of this class with strings in that spec, so I changed this one.